### ISSUE TITLE
Add hyperparameters to learners

### ISSUE THREAD

**Issue by Zukkari** [Zukkari/nirdizati-training-ui#16]
Add hyperparameters to learners. 

Those will be set by default but user will be able to change them based on his preferences.

Based on those settings ```json``` will be generated and used when training the model.

Fields should be added to the training view based on learner method selected.

```
Regression method == Random forest -> n_estimators + max_features

Regression method == Gradient boosting -> max_features + gbm_learning_rate
```

**Comment by verenich** [comment]
this `training_params.json ` file does not need to be persistent, it may be overwritten each time a user submits a training job (using either default or his own settings). It can be as simple as this:
```
{"bpic15": {
      "single_agg": {
         "rf": {
            "max_features": 0.75, 
            "n_estimators": 300
            } 
       } 
   }
}
```

**Comment by verenich** [comment]
you forgot `n_estimators` for gradient boosting
